academic-requirements/src/components/DropTarget.tsx

```diff
@@ -52,10 +52,14 @@ export interface ContainerProps {
     preReq: string;
     category: string;
   }[];
+  CompletedCourses: {
+    identifier: string;
```

Comment:
Does the name "identifier" need to be matched anywhere?

Reply:
No, I just wanted to give it a different name because it is in the subject-number format. However, you bring up a good point and I switched it to just be a list of strings rather than objects.